File: checkout-sdk/src/foam.ts
import * as foam from '@foam-ai/browser-opentelemetry';
import { FOAM_API_KEY, IS_PRODUCTION } from './config';

foam.init({
  serviceName: 'checkout-sdk',
  isProduction: IS_PRODUCTION,
  apiKey: FOAM_API_KEY,
  sampleRate: IS_PRODUCTION ? 0.05 : 1.0,
  captureConsoleErrors: true,
});

File: underwriting-api/src/foam.py
import foam
from config import FOAM_API_KEY, IS_PRODUCTION

foam.init(
    service_name="underwriting-api",
    api_key=FOAM_API_KEY,
    is_production=IS_PRODUCTION,
    sample_rate=1.0 if not IS_PRODUCTION else 0.1,
)